### PR TITLE
Constexpr eval fixes

### PR DESCRIPTION
**Pull request by vlad20012**

### CODE REVIEW

File: src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt [hunk 1]
```diff
@@ -1104,7 +1126,7 @@ private fun RsStubLiteralType?.serialize(dataStream: StubOutputStream) {
         dataStream.writeInt(-1)
```
Comment (Undin) [inline src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt]: `writeInt` -> `writeByte`

Comment (vlad20012) [reply src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt]: Hmm =)

Comment (vlad20012) [reply src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt]: Fixed

**Comment by vlad20012** [comment]
bors try

**Pull request merged**